src/qtiItem/helper/maxScore.js

```diff
@@ -822,5 +822,21 @@ export default {
             max = 0;
         }
         return max;
+    },
+    pciInteraction: function pciInteraction(interaction) {
+        var responseDeclaration = interaction.getResponseDeclaration();
+        var template = responseHelper.getTemplateNameFromUri(responseDeclaration.template);
+        var max;
+        if (template === 'MATCH_CORRECT') {
+            if (
+                _.isArray(responseDeclaration.correctResponse) &&
```

Comment:
**chore:** You can use native API instead of lodash

**suggestion:**
```suggestion
                Array.isArray(responseDeclaration.correctResponse) &&
```